commit 72566e7a58a4e4047aec21775aed9df86bc3a43f
Author: Stefan Wahren <stefan.wahren@i2se.com>
Date:   Sat Nov 18 14:04:13 2017 +0100

    ARM: bcm283x: Add missing interrupt for RNG block
    
    commit 4034600e6f72f7e0a7d8112db3de61469e47fc36 upstream.
    
    This patch adds the missing interrupt property to the RNG block
    of BCM283x.
    
    Link: https://github.com/raspberrypi/linux/issues/2195
    CC: Florian Fainelli <f.fainelli@gmail.com>
    Signed-off-by: Stefan Wahren <stefan.wahren@i2se.com>
    Signed-off-by: Florian Fainelli <f.fainelli@gmail.com>

diff --git a/arch/arm/boot/dts/bcm283x.dtsi b/arch/arm/boot/dts/bcm283x.dtsi
index ac00e730f898..61315cf734ef 100644
--- a/arch/arm/boot/dts/bcm283x.dtsi
+++ b/arch/arm/boot/dts/bcm283x.dtsi
@@ -136,6 +136,7 @@
 		rng@7e104000 {
 			compatible = "brcm,bcm2835-rng";
 			reg = <0x7e104000 0x10>;
+			interrupts = <2 29>;
 		};
 
 		mailbox: mailbox@7e00b880 {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1199_72566e7a58a4e4047aec21775aed9df86bc3a43f.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1199_72566e7a58a4e4047aec21775aed9df86bc3a43f.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
